flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1090,9 +1095,17 @@ private ApplicationReport startAppMaster(
         final ContainerLaunchContext amContainer =
                 setupApplicationMasterContainer(yarnClusterEntrypoint, hasKrb5, processSpec);
 
-        // setup security tokens
+        // Here we have the new and the old delegation token framework together temporarily.
+        // The reason is simple, adding a horror complicated several thousand lines PR is not a good
+        // idea so in the upcoming commits will remove the old DT framework and only the
+        // new will be in place.
+
```

Comment:
I'd remove this comment. If we want a note about this, I'd rather do something along the lines of:

```
// Old delegation token framework (to be removed in FLINK-XXX)

Reply:
OK, removed. It's not required just wanted to give some context for reviewers.